src/engine/TPA.cc

```diff
@@ -405,6 +410,25 @@ VerificationAnswer TPABase::solve() {
     }
 }
 
+VerificationAnswer TPABase::checkTrivialUnreachability() {
+    if (query == logic.getTerm_false()) {
+        // TODO: Check UNSAT with solver?
+        explanation.power = 0;
+        explanation.relationType = TPAType::LESS_THAN;
```

Comment:
Btw, just interesting, why this specific type of relation, can you pls briefly explain this parameters, bc I'm a little bit puzzled?

Reply:
`TPAType::LESS_THAN` should be used so it works for both TPA modes.

The others are probably not necessarily, but should be set to reasonable values so that methods `getSafetyExplanation` and `getInductiveInvariant` work correctly.

Except `power`, which must be set to `0`.
The over-approximation on level 0 is `true` at this point, so fixed point type and invariant type can be anything, but these are a bit safer values. If `query` is `false`, then all transition formulas are invariants when restricted to `query`.
Similarly, if `init` is `false`, then all transition formulas are invariants when restricted to `init`.